### DISCUSSION TITLE
Clarification on changes to GitHub Classroom

### DISCUSSION THREAD

**Discussion by gihanmarasingha** [community/community#79113] [GitHub Education] [answered]
### Hi everyone,

Will the January 2024 changes to GitHub Classroom involve any changes to the Classroom API? Though my students use GitHub Classroom for submission and auto grading, I use the API to download all student repos and then do my own semi-automated grading (using manual marks where necessary).

I want to know if my workflow will be impacted by the change to forked repos.

It would also really help if you produced a versioned specification document for GitHub Classroom. Thanks!

**Comment by RyanHecht** [comment, marked as answer]
Hi @gihanmarasingha,

This change shouldn't affect downloading student repositories with the Classroom CLI or our API routes. I'll be sure to update this thread if I learn otherwise, but you should be good to continue this workflow :)

**Comment by booleanchile** [comment]
Hi! Today I been trying using the new fork based repository for a assigment with autograding tasks, But when the student follow the invitation link, the ".github/workflows" directory is not created.
My workaround was put the ".github/workflows/classroom" and  ".github/workflows/classroom.yml" directory and files manually in the base repository.
